### DISCUSSION TITLE
INBOX Alway have message unread

### DISCUSSION THREAD

**Discussion by Gourds** [community/community#50771] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/13073660/227098112-647a039b-6d90-4b38-809c-215812661971.png)

Obviously there is no message, but the inbox keeps showing messages. As an obsessive-compulsive disorder patient, this is very uncomfortable

**Comment by radar-blip** [comment, marked as answer]
There are many discussions describing the same issue. 
Below are some of those. Try some of the suggestions in the posts below to help fix the issue:

Mainly this one: https://github.com/orgs/community/discussions/6874#discussioncomment-3593547

#6874

#48566

#35381

#35809
